itests/config/additional_settings.go

```diff
@@ -0,0 +1,118 @@
+package config
+
+import (
+	"encoding/json"
+	stderrs "errors"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	testdataFolder = "testdata"
+)
+
+func joinPath(path ...string) string {
```

Comment:
We don't need this function. `filepath.Join` has exactly this signature. See https://go.dev/play/p/9ZfH4MDSVhs

Reply:
Fix.